client/src/components/shared/Info/Info.js

```diff
@@ -83,18 +84,16 @@ const Info = ({data, canClose, onClose}) => {
         <h3 className={`${isPolitician(data) ? 'politician' : ''}`}>
           <Circle aria-hidden="true" />&nbsp;{entity.entity_name}&nbsp;
         </h3>
+        <Link to={`/verejne?lat=${entity.lat}&lng=${entity.lng}&zoom=18`} title="Zobraz na mape">
+          <img src={place} alt="MapMarker" />
+        </Link>
         {canClose && (
           <span className="info-close-button" onClick={onClose}>
             &times;
           </span>
         )}
       </div>
       <div className="info-main">
-        {/*TODO when verejne will be completed link has to be fixed
```

Comment:
This button can be removed too: https://github.com/verejnedigital/verejne.digital/blob/master/client/src/components/Verejne/EntitySearchResultItem/index.js#L33